lib/snippets.js

```diff
@@ -88,6 +88,11 @@ export const get${tableName}ById = async id => {
   return getRecordById(Tables.${tableName}, id);
 };
 
+export const get${tableName}ByIds = async ids => {
```

Comment:
Real nitpick: call `pluralize` here - (you're getting many owners by their IDs, right?)

Reply:
great point